static/src/stylesheets/module/_adslot.scss

```
@@ -103,6 +103,13 @@
     margin: 0 auto;
 }
 
+.ad-slot-container--centre-slot.mobile-banner
+{
+    width: fit-content;
+    padding-bottom: 14px;
+    margin: 0 auto;
+}
+
```

```suggestion
.crossword-mobile-banner-ad
{
    padding-bottom: 14px;
}
```

This can be decoupled.

---

I have modifies the CSS since this review, to add a media query for mobile size:
```css
.ad-slot-container--centre-slot.mobile-banner {
    width: fit-content;
    padding-bottom: 14px;
    margin: 0 auto;
    display: flex;
    justify-content: center;
    @include mq($from: phablet) {
        display: none;
    }
}
```

but will apply the new class name and remove redundant styles.